### PR TITLE
Welcome: fix lead parapgraph, include RC links, and update AotM

### PR DESCRIPTION
**Pull request by MegaApplePi**
- Changed "mania" to "synthesizer" to allow more... consonance, I guess (the last description words end with an -or or an -er sound so synthesizer might make it sound better).
- Included Ranking Criteria links in CPoI.
- Update AotM to Mascots.
- Still need help with the paragraph for it.

## Status

Need help.

### CODE REVIEW

File: wiki/Welcome/en.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+Welcome to osu!, a free-to-win rhythm game developed by peppy with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drumming emulator; osu!catch, a fruit salad catcher; and osu!mania, a key smashing synthesizer.
 
 ## Common points of interest
 
-- [Registration](/wiki/Registration)
 - [Installation](/wiki/Installation)
+- [Registration](/wiki/Registration)
 - [Rules](/wiki/Rules)
 - [FAQ](/wiki/FAQ)
 - Game Modes
```
Comment (Joehuu) [inline wiki/Welcome/en.md]: Link `Game Modes` to https://osu.ppy.sh/help/wiki/Game_Modes.

File: wiki/Welcome/en.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Welcome
 
-Welcome to osu!, a free-to-win rhythm game developed by peppy with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drumming emulator; osu!catch, a fruit salad catcher; osu!mania, a key smashing mania.
+Welcome to osu!, a free-to-win rhythm game developed by peppy with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drumming emulator; osu!catch, a fruit salad catcher; and osu!mania, a key smashing synthesizer.
```
Comment (Joehuu) [inline wiki/Welcome/en.md]: Link `peppy` to https://osu.ppy.sh/users/2.

Comment (MegaApplePi) [reply wiki/Welcome/en.md]: I don't think the lead paragraph for the welcome page should have any links.

File: wiki/Welcome/en.md (hunk 1 above)
Comment (MegaApplePi) [reply wiki/Welcome/en.md]: The Game modes page itself, in its current state, isn't really interesting as it just links to those other pages

**Pull request merged**